index.html

```diff
@@ -4,13 +4,28 @@
     <title>Dollar Drizzle Dayz</title>
     <meta charset="UTF-8" />
     <style>
+      @font-face {
+        font-family: 'Comic Relief';
+        font-weight: normal;
+        font-display: swap;
+        src: url(/fonts/ComicRelief-Regular.woff2) format('woff2');
+      }
+      @font-face {
+        font-family: 'Comic Relief';
+        font-weight: bold;
+        font-display: swap;
+        src: url(/fonts/ComicRelief-Bold.woff2) format('woff2');
+      }
+
       body {
-        background: #000000 url("bg.webp");
+        background: #000000 url(images/bg.webp);
         color: #00ff00;
-        font-family: "Comic Sans MS", cursive;
+        font-family: 'Comic Relief', 'Comic Sans MS', cursive;
```

Comment:
What's the reason to prefer Comic Relief here?

Reply:
A lot of machines, mine included, don't have Comic Sans MS, and the cursive fallback font is very hard to read. We can bundle Comic Relief to ensure the page looks the same to all users.

Reply:
This is how the page looks for me:

![2024-11-15-14-01-59](https://github.com/user-attachments/assets/274e7215-089b-4158-a746-67485fb4682f)

Reply:
I noticed this on my phone this morning. I think we should prefer Comic Sans for the authentic Geocities experience and use Comic Relief as the fallback. Wdyt?